Dockerfile

```diff
@@ -39,7 +39,7 @@ ENV LC_ALL="en_US.UTF-8"
 # INSTALL NODE, GIT & GO
 RUN add-apt-repository ppa:git-core/ppa --yes \
  && add-apt-repository ppa:longsleep/golang-backports --yes \
- && curl --location https://deb.nodesource.com/setup_12.x | bash \
+ && curl --location https://deb.nodesource.com/setup_14.x | bash \
```

Comment:
why the version bump?

Reply:
My bad, I messed the branch I was testing this after the node debacle